haskell/test/Test/Ligo/BaseDAO/Common.hs

```diff
@@ -46,7 +47,7 @@ import Ligo.BaseDAO.Types
 import Test.Ligo.BaseDAO.Common.StorageHelper as StorageHelper
 import Test.Ligo.BaseDAO.Proposal.Config (ConfigDesc, fillConfig)
 
-type OriginateFn m = m DaoOriginateData
+type OriginateFn m = QuorumThreshold -> m DaoOriginateData
```

Comment:
This was required because there is no `quorumThreshold` field in the `config`, and in tests we use `config` to pass initialization values to the origination function. Till now, we had used a hack by using`minQuorumThreshold` value in config to initialize `quorumThreshold` in storage during contract origination. But this caused some unexpected behavior that I encountered. This caused the quorum theshold update tests to fail, when I moved them to nettest interface, because both the initial quorum threhold and the `minQuorumThreshold` was set to the same value, and thus it could never go below the initial value during updates.

Reply:
Perhaps we should stop using the config-based approach and instead make up a datatype for configuring in tests :thinking: 

I think we can leave this to be decided in #55